deepface/modules/preprocessing.py

```diff
@@ -34,7 +34,7 @@ def load_image(img: Union[str, np.ndarray]) -> Tuple[np.ndarray, str]:
         return load_base64(img), "base64 encoded string"
 
     # The image is a url
-    if img.startswith("http"):
+    if img.startswith("http://") or img.startswith("https://"):
```

Comment:
AFAIK python's string comparison is case sensisitve. Hance this might miss upper case `HTTP://` and all combinations in between. Maybe transform to lower case before ? (as you seem to don't like regexes)

Reply:
morning, you are absolutely right. cheers!

creating an issue to follow this.